### ISSUE TITLE
alpacascope quits moments after starting up

### ISSUE THREAD

**Issue by wbendick** [synfinatic/alpacascope#107]
Is there a log saved somewhere so I can try to figure out why?

**Comment by synfinatic** [comment]
there is a `--debug` flag.  That might provide you some info, but probably not.   Are you running Windows?  Do you have the appropriate OpenGL driver installed for your video card?

**Comment by wbendick** [comment]
Yes, I am running Windows. I think opengl is working fine.. video games run fine. alpacascope used to work on this computer but stopped some months ago and I am trying to figure it out now. --debug showed nothing

**Comment by synfinatic** [comment]
Most games are DirectX and not OpenGL so that doesn’t mean much.  Please reinstall OpenGL. -AaronOn Jul 31, 2024, at 11:27 PM, Will Bendick ***@***.***> wrote:﻿
Yes, I am running Windows. I think opengl is working fine.. video games run fine. alpacascope used to work on this computer but stopped some months ago and I am trying to figure it out now. --debug showed nothing

**Comment by wbendick** [comment]
I don't think OpenGL is the problem. If I start the app without ascom remote server running, alpacascope doesn't quit. It just shows error messages about not being able to find ascome remote server.

**Comment by synfinatic** [comment]
I'm assuming you're running Windows?  Have you tried the CLI version?  https://github.com/synfinatic/alpacascope/releases/download/v2.4.0/AlpacaScope-CLI-2.4.0.exe

Failing that, more details about your system/setup?   This GUI version doesn't log to the console.

**Comment by wbendick** [comment]
When I run AlpacaScope-CLI-2.4.0.exe I see:

ERRO[0000] found ips: All-Interfaces/0.0.0.0, Bluetooth Network Connection/169.254.1.5, Bluetooth Network Connection/fe80::c619:44cd:3db9:e53d, Ethernet 2/192.168.86.65, Ethernet 2/fdf4:c0ff:48ce:4f99:1e9f:2b19:a0e9:8b43, Ethernet 2/fdf4:c0ff:48ce:4f99:5527:ec45:6709:d6be, Ethernet 2/fe80::2da6:a8d9:3bed:8bb0, Ethernet/169.254.197.79, Ethernet/fe80::95de:9e04:e701:3a2c, Local Area Connection* 1/169.254.171.83, Local Area Connection* 1/fe80::9436:57bd:ae3d:efe8, Local Area Connection* 10/169.254.200.104, Local Area Connection* 10/fe80::246:3819:91d2:a1fa, Loopback Pseudo-Interface 1/127.0.0.1, Loopback Pseudo-Interface 1/::1, Wi-Fi/169.254.70.89, Wi-Fi/fe80::b8ff:cf4d:f686:118d
INFO[0000] Looking for Alpaca Remote Server locally on port 11111...
INFO[0001] Found Alpaca on 192.168.86.65:11111
INFO[0001] Starting SkyFi Discovery service on UDP/4031
FATA[0001] Telescope is not connected to ASCOM Remote

This last log line does not make sense to me because in ASCOM Remote Server Console I see:

Device Telescope 0 (EQMOD.Telescope) connected OK and is available
ASCOM Remote Alpaca device listening URI: http://127.0.0.1:11111/api/
Listening for IPv4 discovery broadcasts on port 32227.
IPv6 discovery not enabled.
Server started successfully.

**Comment by synfinatic** [comment]
Can you run with the `--debug` flag?

**Comment by wbendick** [comment]
C:\Users\wbend\Downloads>AlpacaScope-CLI-2.4.0.exe --debug
time="2024-08-10T15:02:08-07:00" level=error msg="found ips: All-Interfaces/0.0.0.0, Bluetooth Network Connection/169.254.1.5, Bluetooth Network Connection/fe80::c619:44cd:3db9:e53d, Ethernet 2/192.168.86.65, Ethernet 2/fdf4:c0ff:48ce:4f99:1e9f:2b19:a0e9:8b43, Ethernet 2/fdf4:c0ff:48ce:4f99:5527:ec45:6709:d6be, Ethernet 2/fe80::2da6:a8d9:3bed:8bb0, Ethernet/169.254.197.79, Ethernet/fe80::95de:9e04:e701:3a2c, Local Area Connection* 1/169.254.171.83, Local Area Connection* 1/fe80::9436:57bd:ae3d:efe8, Local Area Connection* 10/169.254.200.104, Local Area Connection* 10/fe80::246:3819:91d2:a1fa, Loopback Pseudo-Interface 1/127.0.0.1, Loopback Pseudo-Interface 1/::1, Wi-Fi/169.254.70.89, Wi-Fi/fe80::b8ff:cf4d:f686:118d"
time="2024-08-10T15:02:08-07:00" level=info msg="Looking for Alpaca Remote Server locally on port 11111..."
time="2024-08-10T15:02:08-07:00" level=debug msg="local addrs: [fe80::95de:9e04:e701:3a2c/64 169.254.197.79/16 fdf4:c0ff:48ce:4f99:1e9f:2b19:a0e9:8b43/64 fdf4:c0ff:48ce:4f99:5527:ec45:6709:d6be/128 fe80::2da6:a8d9:3bed:8bb0/64 192.168.86.65/24 fe80::b8ff:cf4d:f686:118d/64 169.254.70.89/16 fe80::9436:57bd:ae3d:efe8/64 169.254.171.83/16 fe80::246:3819:91d2:a1fa/64 169.254.200.104/16 fe80::c619:44cd:3db9:e53d/64 169.254.1.5/16 ::1/128 127.0.0.1/8]"
time="2024-08-10T15:02:09-07:00" level=debug msg="Alpaca is not running on 169.254.197.79:11111"
time="2024-08-10T15:02:09-07:00" level=info msg="Found Alpaca on 192.168.86.65:11111"
time="2024-08-10T15:02:09-07:00" level=info msg="Starting SkyFi Discovery service on UDP/4031"
time="2024-08-10T15:02:09-07:00" level=debug msg="Alpaca response: (*alpaca.boolResponse)(0xc000120c80)({\n Value: (bool) false,\n ClientTransactionID: (uint32) 0,\n ServerTransactionID: (uint32) 0,\n ErrorNumber: (int32) 0,\n ErrorMessage: (string) \"\"\n})\n"
time="2024-08-10T15:02:09-07:00" level=debug msg="Alpaca PUT: (map[string]string) (len=3) {\n (string) (len=9) \"Connected\": (string) (len=4) \"true\",\n (string) (len=8) \"ClientID\": (string) (len=10) \"2596996162\",\n (string) (len=19) \"ClientTransactionID\": (string) (len=1) \"2\"\n}\n"
time="2024-08-10T15:02:09-07:00" level=debug msg="Alpaca PUT response: (*alpaca.putResponse)(0xc0000880a0)({\n ClientTransactionID: (uint32) 0,\n ServerTransactionID: (uint32) 0,\n ErrorNumber: (int32) 0,\n ErrorMessage: (string) \"\"\n})\n"
time="2024-08-10T15:02:09-07:00" level=debug msg="Alpaca response: (*alpaca.boolResponse)(0xc000088100)({\n Value: (bool) false,\n ClientTransactionID: (uint32) 0,\n ServerTransactionID: (uint32) 0,\n ErrorNumber: (int32) 0,\n ErrorMessage: (string) \"\"\n})\n"
time="2024-08-10T15:02:09-07:00" level=fatal msg="Telescope is not connected to ASCOM Remote"

C:\Users\wbend\Downloads>

**Comment by synfinatic** [comment]
well, the debug logs match the output.   Not sure if you're seeing anything in the ASCOM Remote console.  What version of ASCOM Remote are you running?   

Gonna take me a bit to be able to reproduce/debug on my end further since I'm in the process of re-organizing my office/new desk and my Windows PC is packed away somewhere.

**Comment by wbendick** [comment]
ASCOM Remote Server v6.7.1 (Build 4785) - 32bit mode.

**Comment by synfinatic** [comment]
So in ASCOM Remote, click the "Log Requests" and "Log Responses" boxes and try again.  And paste in the results.  Honestly, the issue seems to be your telescope is not responding/configured correctly.

BTW, it would help if you use the "code" formatting tool for the logs.  would make it more readable.  thanks.
